src/ResourceManager/TrafficManager/Commands.TrafficManager2/help/New-AzureRmTrafficManagerProfile.md

```diff
@@ -42,6 +46,19 @@ The DNS FQDN is contosoapp.trafficmanager.net.
 
 ## PARAMETERS
 
+### -CustomHeaders
+List of custom header name and value pairs for probe requests.```yaml
```

Comment:
This is one of the trailing yaml strings mentioned in the validatehelpissues report

Reply:
Yes, I see them in 3 files. Will fix.

Reply:
Fixed.